src/components/Editor/SearchBar.js

```diff
@@ -163,7 +163,11 @@ class SearchBar extends Component {
       searchInput.focus();
     }
 
-    if (this.refs.resultList && this.refs.resultList.refs) {
+    if (
+      this.refs.resultList &&
+      this.refs.resultList.refs &&
+      this.state.symbolSearchResults.length > this.state.selectedResultIndex
```

Comment:
This can also be implemented as
```javascript
this.refs.resultList.refs.hasOwnProperty(this.state.selectedResultIndex)
```
Or adding condition in https://github.com/devtools-html/debugger.html/blob/master/src/utils/result-list.js#L4 
```javascript
function scrollList(resultList, index) {
  if (!resultList.hasOwnProperty(index)) return;
  const resultEl = resultList[index];
```